include/deal.II/lac/trilinos_solver.h

```diff
@@ -1177,7 +1177,7 @@ namespace TrilinosWrappers
      * operators/preconditioners. For details, see
      * https://docs.trilinos.org/latest-release/packages/belos/doc/html/classBelos_1_1Operator.html.
      */
-    template <class OperatorType, class VectorType>
+    template <class OperatorType, typename VectorType>
```

Comment:
```suggestion
    template <typename OperatorType, typename VectorType>
```

Reply:
Let me do this in a follow-up, assuming that the test suite runs to completion successfully.